ooiui/static/css/common/toc_menu.css

```diff
@@ -2,8 +2,14 @@
     max-height:80%;
 }
 #assetBrowser {
-    margin-top:25px;
```

Comment:
@birdage please checked this doesn't do anything strange on the data catalog page.